markdown_lab/core/converter.py

```
@@ -380,26 +380,33 @@ def _process_single_url(
-                url_chunk_dir = f"{chunk_dir}/{Path(filename).stem}"
-                chunker = ContentChunker(
-                    self.config.chunk_size, self.config.chunk_overlap
-                )
-                chunker.save_chunks(chunks, url_chunk_dir, chunk_format)
+    def _generate_output_filename(self, url: str, output_format: str, output_path: Path) -> str:
+        """Generate the full output file path for a URL."""
+        filename = self.get_filename_from_url(url, output_format)
+        return str(output_path / filename)
+
+    def _save_content_chunks(
+        self, 
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)